suite/tests/CMakeLists.txt

```diff
@@ -482,12 +482,12 @@ endfunction(append_link_flags)
 # line above and instead selectively remove -O3 from tests that cannot
 # support it.
 # This was investigated and proved to be non trivial as the client tests
-# use _DR_set_compile_flags() to set the compile flags whereas the core 
+# use _DR_set_compile_flags() to set the compile flags whereas the core
 # tests use set_cflags(). _DR_set_compile_flags() is a public function and we
 # didn't want to add a blocklist in there.
 # For now we create a list of tests that can be built with -O3.
 # This is for AARCH64 UNIX only.
-# TODO change this allowlist to a blocklist
+# TODO i#6429 Optimization flags are removed for all tests.
```

Comment:
I would keep your original text: `TODO i#6429: Change this allowlist to a blocklist.`